doc/source/index.rst

```diff
@@ -1,9 +1,33 @@
-Welcome to SUSE Containerized OpenStack
-=======================================
+Welcome to SUSE Containerized OpenStack Technology Preview
+==========================================================
 
-The socok8s project automates SUSE Containerized OpenStack provisioning
-and lifecycle management on SUSE Container as a Service Platform (CaaSP) and
-SUSE Enterprise Storage (SES), using Airship, shell scripts, and Ansible playbooks.
+The socok8s project automates SUSE Containerized OpenStack provisioning and
+lifecycle management on SUSE Container as a Service Platform (CaaSP) and
+SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
+and Ansible playbooks.
+
+As a technology preview, the included features are not supported. They may be
+functionally incomplete, unstable or in other ways not suitable for production
+use. They are mainly included for customer convenience and give customers a
+chance to test new technologies within an enterprise environment.
+
+This release deploys the following set of core OpenStack services:
+
+  * Keystone
+  * Glance
+  * Cinder
+  * Nova
+  * Neutron
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later depends on customer
+and market feedback. A technology preview does not automatically result in
+support at a later point in time. It could be dropped at any time and SUSE is not
+committed to providing a technology preview later in the product cycle.
+
+Please, give your SUSE representative feedback, including your experience and
+use case.
```

Comment:
Replace
Please provide your feedback using the beta mailing lists that are documented on the technology preview download page or your SUSE representative.  We are very interested in your experience and use case.